src/Plugins/ComplexCore/test/RawBinaryReaderTest.cpp

```diff
@@ -0,0 +1,148 @@
+/**
```

Comment:
Remove this entire comment block if the test is running correctly. You may want to add a concise statement about what use cases this unit test covers